cogs/vxtwitconverter/eventsCore.py

```diff
@@ -0,0 +1,42 @@
+from discord import Message, channel
+from urlextract import URLExtract
+
+from cogs.vxtwitconverter.core import Core
+
+
+class EventCore(Core):
```

Comment:
Can this be changed to `EventsCore` to match others?
```suggestion
class EventsCore(Core):
```